templates/stats.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{% import "macros.html" as macros %}
+{% block title %}Koschei - statistics{% endblock %}
+{% block content %}
+<div class="pageHeader">Koschei in numbers</div>
+<table class="details">
+    <tr><th>Stats update time</th><td>{{ stats.refresh_time }}</td></tr>
+    <tr><th>All packages</th><td>{{ stats.packages }}</td></tr>
+    <tr><th>Tracked packages</th><td>{{ stats.tracked_packages }}</td></tr>
+    <tr><th>Untracked packages</th><td>{{ stats.packages - stats.tracked_packages }}</td></tr>
+    <tr><th>Blocked packages</th><td>{{ stats.blocked_packages }}</td></tr>
+    <tr><th>Active packages</th><td>{{ stats.packages - stats.blocked_packages }}</td></tr>
+    <tr><th>All builds (real and scratch)</th><td>{{ stats.builds }}</td></tr>
+    <tr><th>Real builds</th><td>{{ stats.real_builds }}</td></tr>
```

Comment:
This makes me realize that we chose the term "real" kinda arbitrarily and it might not be that obvious what it means. Can you add some short explanation?
(I should add that to package_detail as well)